search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -91,10 +95,77 @@ function displayTopResultsForCurrentTrend(countryCode) {
     // Get search results for the specified country.
     let results = countryData[0].results;
     for (let i = 0; i < results.length; i++) {
-      resultElement.innerHTML += '<a href=' + results[i].link + '>' +
-        results[i].htmlTitle + '</a><br>' + results[i].snippet + '<br>'
-        + '<i>Sentiment Score: ' + results[i].score.toFixed(1) + '</i><br>';
+      // i + 1 shows the index for each search result. 
+      resultElement.innerHTML += (i + 1).toString() + '. ' + '<a href=' + 
+          results[i].link + '>' + results[i].htmlTitle + '</a><br>' + 
+          results[i].snippet + '<br>' + 'Sentiment Score: ' + results[i].score + 
+          '<br>';
     }
   }
   resultElement.innerHTML += '<i>Last updated on ' + date + '</i>';
 }
+
+/** 
+ * Displays sentiment chart of search results for the current country on the modal.
+ * @param {string} countryCode Two letter country code for selected country.
+ */
+function displaySentimentChartForCurrentTrend(countryCode) {
+  // let topic = getCurrentTrend();
+  let dataByCountry = getCurrentCustomSearchData().dataByCountry;
+  let countryData = dataByCountry.filter(data => data.country === countryCode);
+  // let topicData = getCurrentCustomSearchData();
+  let date = new Date(getCurrentCustomSearchData().timestamp);
+  let chartElement = document.getElementById('sentiment-chart-tab');
+  chartElement.innerHTML = '';
+
+  // Handle case where there are no results.
+  if (countryData.length === 0 ||
+      countryData[0].averageSentiment === NO_RESULTS_DEFAULT_SCORE) {
+    chartElement.innerHTML += 'No results.<br><i>Last updated on ' +
+        date.toString() + '<i><br>';
+  } else {
+    let results = countryData[0].results;
+    drawSentimentChart(chartElement, results);
+    chartElement.innerHTML += 'Average Sentiment Score: ' + 
+        countryData[0].averageSentiment + '<br>';
+    chartElement.innerHTML += '<i>Last updated on ' + date.toString() +
+        '<i><br>';
+  }
+}
+
+google.charts.load('45', {'packages':['corechart']});
```

Comment:
What does 45 mean here? Can you add a comment

Reply:
Also can you add a comment for why this needs to be outside the function?